### DISCUSSION TITLE
How to use the tab-set instruction?

### DISCUSSION THREAD

**Discussion by VForiel** [executablebooks/MyST-Parser#692] [Q&A] [answered]
The documentation uses tab-set instructions, but they are not documented and I don't find how to use them

https://github.com/executablebooks/MyST-Parser/blob/84c320b27d049d9a223c88b50ca8cf159e7707b6/docs/syntax/optional.md?plain=1#L306-L336

When I copy these instructions on my side it say that these instructions are unknown. Is it managed by another plugin? If so, which one?

**Comment by chrisjsewell** [comment, marked as answer]
Yep sphinx-design: https://sphinx-design.readthedocs.io/en/furo-theme/
